apps/web/pages/export.tsx

```diff
@@ -119,110 +123,59 @@ function SyncSpreadsheetCard(props: SyncSpreadsheetCardProps) {
-              </li>
-              {/* negative margin-left to offset padding of the ordered list */}
-              <div className="-ml-4 flex gap-2 py-2">
-                <input
-                  className="h-8 grow truncate rounded-lg bg-venice-black-400 px-2 font-mono text-xs text-venice-gray/75 ring-1 ring-inset ring-venice-black-300 focus:outline-none"
-                  value={googleSheetImport}
-                  readOnly
-                />
-                <CopyTextButton content={googleSheetImport} />
-              </div>
-              <li>
-                There is no step 2! Congratulations, your spreadsheet will now
-                always be up-to-date with Venice!
-              </li>
-            </ol>
-          </section>
-          <section className="grid gap-2">
-            <p>
-              <span className="text-venice-green">(2 minutes)</span> Microsoft
-              Excel with manual refresh:
-            </p>
-            <p className="pl-4">
-              Using Excel, you can treat a Venice CSV file as a local
-              &quot;source of truth&quot; which refreshes your spreadsheet every
-              time that CSV file is overwritten.{' '}
-              <button
-                className="text-venice-green hover:text-venice-green-darkened disabled:cursor-default disabled:hover:text-venice-green"
-                disabled={isEmptyResult}
-                onClick={() => window.open(csvQuery.downloadUrl)}>
-                Download your data as a CSV
-              </button>{' '}
-              file and then{' '}
-              <ExternalLink
-                href="https://docs.venice.is"
-                className="text-venice-green hover:text-venice-green-darkened">
-                read our docs
-              </ExternalLink>{' '}
-              for more info!
-            </p>
-          </section>
-        </div>
-      </div>
-    </Card>
-  )
-}
-
-interface CopyTextButtonProps {
-  content: string
-}
-
-function CopyTextButton(props: CopyTextButtonProps) {
-  const [isCopied, setCopied] = useState(false)
-  const {icon: Icon, text} = isCopied
-    ? {icon: CheckCircleFilledIcon, text: 'Copied'}
-    : {icon: CopyTextIcon, text: 'Copy'}
-
-  async function copyToClipboard() {
-    if (navigator) {
-      try {
-        await navigator.clipboard.writeText(props.content)
-        setCopied(true)
-      } catch (err) {
-        // TODO report via Sentry
-        console.error('Unable to copy content to clipboard.', err)
-      }
-    }
-  }
-
-  useEffect(() => {
-    if (isCopied) {
-      const timer = setTimeout(() => setCopied(false), 2000)
-      return () => clearTimeout(timer)
-    }
-    return
-  }, [isCopied])
-
-  return (
-    <Button
-      variant="primary"
-      disabled={isCopied}
-      onClick={copyToClipboard}
-      className="shrink-0 gap-1 disabled:opacity-100">
-      <Icon className="h-4 w-4 fill-current" />
-      <span>{text}</span>
-    </Button>
+    <InstructionCard
```

Comment:
See `packages/ui/components/InstructionCard.tsx`—extract a common design for this type of Card.